src/mcts/node.h

```diff
@@ -165,6 +165,7 @@ class Node {
   // Returns whether the node is known to be draw/lose/win.
   bool IsTerminal() const { return terminal_type_ != Terminal::NonTerminal; }
   bool IsTbTerminal() const { return terminal_type_ == Terminal::Tablebase; }
+  bool IsTwofoldTerminal() const { return terminal_type_ == Terminal::TwoFold; }
```

Comment:
Isn't IsTwoFoldTerminal more consistent?

Reply:
You are right. Corrected it.